website/views/shared/modals/members.jade

```diff
@@ -46,9 +46,12 @@ script(type='text/ng-template', id='modals/private-message.html')
   .modal-header
     h4=env.t('pmHeading', {name: "{{profile.profile.name}}"})
   .modal-body
-    textarea.form-control(type='text',rows='5',ui-keydown='{"meta-enter":"sendPrivateMessage(profile._id, _message)"}',ng-model='_message')
+    textarea.form-control(type='text',rows='5',ui-keydown='{"meta-enter":"sendPrivateMessage(profile._id, _message)"}',placeholder=env.t('needsTextPlaceholder'),ng-model='_message')
```

Comment:
@rdhammond Just want to double check two things:

1) You're on a windows machine using Chrome, right? (On a mac, you'd do `cmd + enter`)
2) Does the `ctrl + enter` just not work on this branch? Or does it not work on the website as well?

I don't have access to a PC right now. Can another dev confirm the behavior of `ctrl + enter` when sending PMs?